### PR TITLE
Rename pleo-oss to pleo-io

### PR DESCRIPTION
**Pull request by jsfr**
Removed all mentions of pleo-oss

### CODE REVIEW

**Review by marcos-arranz — APPROVED** [review]

**Review by andersfischernielsen — COMMENTED** [review]
N/A

**Review by andersfischernielsen — CHANGES_REQUESTED** [review]
N/A

File: README.md [hunk 1]
```diff
@@ -163,4 +163,4 @@ For more, check out the [Contributing Guide](CONTRIBUTING.md).
 
 ## License
 
-[Unlicense](LICENSE) © 2022 pleo-oss
+[Unlicense](LICENSE) © 2022 pleo-io
```
Comment (andersfischernielsen) [inline README.md:166]: ... except for this change, nice catch!

**Review by andersfischernielsen — APPROVED** [review]
I get these changes now - LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 #### 🏠 Internal
 
-- Prefer 'pleo-io' to 'pleo-oss' [#119](https://github.com/pleo-io/file-distributor/pull/119) ([@andersfischernielsen](https://github.com/andersfischernielsen))
+- Prefer 'pleo-io' to 'pleo-io' [#119](https://github.com/pleo-io/file-distributor/pull/119) ([@andersfischernielsen](https://github.com/andersfischernielsen))
```
Comment (andersfischernielsen) [inline CHANGELOG.md:34]:
```suggestion
- Prefer 'pleo-io' to 'pleo-oss' [#119](https://github.com/pleo-io/file-distributor/pull/119) ([@andersfischernielsen](https://github.com/andersfischernielsen))
```

**Pull request merged**